src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -4,22 +4,10 @@
 
 public class ColorSupplier {
     public String getRandomColor() {
-        Random rnd = new Random();
-        switch (rnd.nextInt(100) % 7) {
-            case 0:
-                return "red";
-            case 1:
-                return "orange";
-            case 2:
-                return "yellow";
-            case 3:
-                return "green";
-            case 4:
-                return "blue";
-            case 5:
-                return "violet";
-            default:
-                return "black";
-        }
+        final int colorNumber = 7;
+        final int quasy = 41;
+        String [] colorArray = {"Red", "Orange", "Green", "Yellow", "Blue", "Violet", "Black"};
+        Random random = new Random();
+        return colorArray[random.nextInt(quasy) % colorNumber];
```

Comment:
we can simplify this line to:
```suggestion
        return colorArray[random.nextInt(colors.length)];
```